src/test/java/org/gridsuite/study/server/service/StudyServiceDynamicSimulationTest.java

```diff
@@ -205,4 +213,22 @@ public void testGetDynamicSimulationMappings() throws JsonProcessingException {
         getLogger().info("Mapping infos result in Json = " + objectMapper.writeValueAsString(mappingInfos));
         assertEquals(MAPPINGS.size(), mappingInfos.size());
     }
+
+    @Test
+    public void testDeleteResults() {
```

Comment:
You test nothing ?

Reply:
This part wasn't well well thought out I got a bit lost in since the dynamic simulation isn't tested the same way as the other computations, I've reworked it [here](https://github.com/gridsuite/study-server/pull/429/files#diff-1666c5f0905b6ba072ed5725fd856b1df09f364415d7eaf60b923007161fbf16R407)